vertx-sql-client/src/main/java/io/vertx/sqlclient/impl/SqlClientBase.java

```diff
@@ -37,7 +37,13 @@
 import java.util.function.Function;
 import java.util.stream.Collector;
 
-public abstract class SqlClientBase<C extends SqlClient> implements SqlClient, CommandScheduler {
+public abstract class SqlClientBase<C extends SqlClient> implements SqlClientInternal, CommandScheduler {
+
+  @Override
+  public int appendQueryPlaceholder(StringBuilder queryBuilder, int index, int current) {
+    queryBuilder.append("?");
```

Comment:
we need to override this method in `MSSQLConnectionImpl` and `MSSQLPoolImpl` for sql server client.
The syntax is like postgres as:
```
    queryBuilder.append('@').append(1 + index);
    return index;
```

Reply:
oh good to know I will do it